benchmark/README.md

```diff
@@ -1,41 +1,99 @@
-Swift Benchmark Suite
-=====================
 
-This directory contains the Swift Benchmark Suite.
+# Swift Benchmark Suite
 
-Running Swift Benchmarks
-------------------------
+This directory contains the Swift Benchmark Suite. This is meant to be a suite
+of regression, micro-benchmark, small, and medium sized benchmarks that is used
```

Comment:
Recommended change to "of regression, micro, small, and medium sized benchmarks that are used"